scalafmt-tests/src/test/resources/newlines/alwaysBeforeTopLevelStatements.source

```diff
@@ -36,3 +36,16 @@ package b {
   import a.b.c
   class d {}
 }
+<<< #mytest
+package a.b.c
+package d.e
+package f
+import g.h.i.j
+package k {}
+>>>
+package a.b.c
+package d.e
+package f
+
+import g.h.i.j
+package k {}
```

Comment:
should there be an empty line here as well? perhaps it was this way before.

Reply:
Currently, scalafmt does not change formatting between import and class and keeps an empty line (if it present) as is.